runners/flink/job-server/flink_job_server.gradle

```diff
@@ -94,6 +94,7 @@ dependencies {
   runtimeOnly project(":sdks:java:io:google-cloud-platform")
   // SqlTransform
   runtimeOnly project(":sdks:java:extensions:sql:expansion-service")
+  permitUnusedDeclared project(":runners:flink:1.8")
```

Comment:
Didn't you already do this in `runners/flink/1.8/job-server/build.gradle`?

Reply:
removed it from here.